### PR TITLE
Update versions.list

### PR DESCRIPTION
**Pull request by segiddins**
\c @indirect @fotanus

### CODE REVIEW

File: script/integration/test.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 names = Set.new
 
 #url = "bundler-api-staging.herokuapp.com"
-url = "localhost:9292"
+url = "localhost:5000"
```
Comment (fotanus) [inline script/integration/test.rb]: You meant to commit this? :+1: for the rest

Comment (segiddins) [inline script/integration/test.rb]: yeah, not that it really matters

**Comment by indirect** [comment]
info ALL the checksums

**Comment by indirect** [comment]
ugh I forgot you added homu, sorry

File: spec/update/job_spec.rb [hunk 2]
```diff
@@ -88,16 +89,24 @@ def dependencies(name, version = "1.0", platform = "ruby")
...
+        last_response.body
+        checksum = Digest::MD5.hexdigest(last_response.body)
+
+        database_checksum = get_gem(:info_checksum, db, 'foo1', version)[:info_checksum]
+        expect(database_checksum).to eq(checksum)
+      end
+
+      expect(db[:rubygems]
+        .join(:versions, rubygem_id: :id)
+        .where(name: "foo1").order_by(:created_at).map { |v| v[:number] }).to eq(versions)
```
Comment (sonalkr132) [inline spec/update/job_spec.rb]: Why did you add this here? Test says that `BundlerApi::Job saves checksum for info endpoint content`, when this expect block tests whether all the versions exist in database. :confused:
